docs/building-apps/ingest-sdk/overview.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Overview
+sidebar_position: 10
+---
+
+This tutorial walks through how an application can leverage common streaming data patterns to ingest Stellar network transaction data using a few select packages from the Stellar Go Repo [github.com/stellar/go](https://github.com/stellar/go/blob/master/) collectively known as the 'Ingestion' SDK:
+
+## The Ingestion SDK packages
+
+- `github.com/stellar/go/amount` utility package to convert prices from network transaction operations to string
+- `github.com/stellar/go/historyarchive` `github.com/stellar/go/support/datastore` `github.com/stellar/go/support/storage` utility package with convenient wrappers for accessing history archives, and avoid low-level http aspects.
+- `github.com/stellar/go/ingest` provides parsing functionality over the network ledger data, converts to more developer centric `LedgerTransaction` model
+- `github.com/stellar/go/ingest/ledgerbackend` provides the captive core backend implementation
+- `github.com/stellar/go/network` provides convenient pre-configured settings for testnet and pubnet networks.
+- `github.com/stellar/go/xdr` a complete golang binding to the stellar network data model.
+
+## Ingestion project setup
+
+### Project requirements
+
+To build an example streaming network ingestion pipeline from live Stellar network transaction data, you'll need:
+
+- A developer workstation with [Go](https://go.dev/learn/) programming language runtime installed.
+- An IDE to edit Go code, [VSCode](https://code.visualstudio.com/download) is good if one needed.
```

Comment:
Should ZeroMQ be listed as a requirement here?

Reply:
the context here is more focused on just ingestion sdk, which doesn't have relation/require to ZeroMQ, that part was only specific within the code example as an optional event driven micro-service.